File: AITTimeShowJava/app/src/main/java/hu/ait/aittimeshowjava/MainActivity.java
package hu.ait.aittimeshowjava;

import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;
import android.widget.Toast;

import java.util.Date;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        Button btnTime = findViewById(R.id.btnTime);

        btnTime.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                Toast.makeText(MainActivity.this,
                        new Date(System.currentTimeMillis()).toString(), Toast.LENGTH_SHORT).show();
            }
        });
    }
}

File: AITHelloCafeteria/app/src/main/java/hu/ait/aithellocafateria/MainActivity.kt
package hu.ait.aithellocafateria

import android.support.v7.app.AppCompatActivity
import android.os.Bundle
import android.widget.Toast
import kotlinx.android.synthetic.main.activity_main.*
import java.util.*

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        btnCafateria.setOnClickListener {
            tvStatus.text= "Best cafeteria: Graphisoft main cafeteria"
            Toast.makeText(this, Date(System.currentTimeMillis()).toString(),
                Toast.LENGTH_LONG).show()
        }

    }
}
